css/style.css

```diff
@@ -0,0 +1,998 @@
+}
+.block .block-panel-right,
+.block .block-panel-left {
+  width: 50%;
+}
+.block .block-panel-right p {
+  margin-bottom: 40px;
+}
+
+/*................................................
+    4.SKELETON
+.................................................*/
+#header {
+  padding: 25px 0;
+}
+main {
+  margin-bottom: 100px;
+}
+#footer {
+  padding-top: 70px;
+}
+/*................................................
+    5.HEADER SECTION
+.................................................*/
+#header .nav .flex-box ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: white;
+}
+.flex-box li a {
+  display: block;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font: 16px "Manrope-Medium";
+}
+
+/*................................................
+   6.BODY SECTION 
+.................................................*/
+.banner .container {
+  align-items: center;
+}
+.banner .container .panel-left,
+.banner .container .panel-right {
+  width: 50%;
+}
+.banner .container .panel-left h2, p {
+  margin-bottom: 20px;
+}
+.block .block-panel-right h3, p{
+  text-align: left;
+}
+.block .block-panel-right h5 {
+  text-align: left;
+  display: grid;
+}
+.block ul {
+  display: flex;
+}
+.listing-style-one li {
+  width: 33.333%;
+  padding: 0 20px;
+}
+.listing-style-one li p, h4 {
+  text-align: center;
+}
+.listing-style-one h4 {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  color: #3a3a3a;
+}
+
+.listing-style li img {
+  border-radius: 50%;
+  background: #f0f1ec;
+  padding: 20px;
+  max-width: 100px;
+  vertical-align: middle;
+} 
+.listing-style-two h4 {
+  text-align: left;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  color: #3a3a3a;
+}
+.listing-style-two {
+  flex-wrap: wrap;
+}
+.listing-style-two li {
+  flex: 0 1 33.33%;
+  padding: 20px;
+  display: grid;
+}
+.listing-style-two li:hover {
+  background-color: #fff;
+  border-radius: 12px;
+}
+.listing-style-three li {
+  flex: 33.33%;
+  padding: 20px;
+}
+.listing-style-three {
+  flex-wrap: wrap;
+} 
+.listing-style-three li img {
+  width: 100%;
+  object-fit: cover;
+  height: 100%;
+  vertical-align: middle;
+}
+.flex-box .slider-info img {
+  border-radius: 50%;
+  background: #709d61;
+  padding: 10px;
+  max-width: 50px;
+  height: 50px;
+  margin-bottom: 8px;
+}
+.block .flex-box img {
+  max-width: 300px;
+}
+.slider-info h4 {
+  text-align: left;
+  font: 20px "Manrope-Bold";
+  color: #000000;
+}
+
+.flex-box .slider-info {
+  padding-left: 80px;
+}
+.letter-box {
+  background-color: #f1f2ee;
+  border-radius: 10px;
+  padding: 50px 15px;
+}
+.block-center {
+  text-align: center;
+}
+.slider-info {
+  width: 60%;
+}
+.block .title-area {
+  border-bottom: 30px solid #f7f6f4;
+}
+.mail-box {
+  border-radius: 10px;
+  padding: 10px;
+  max-width: 550px;
+  margin: 0 auto;
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+.letter-box .mail-box
```

Comment:
Selector declared without any CSS properties. Remove unused selector. Please make sure you read all review comments properly and make changes accordion to them.